concent_api/utils/api_view.py

```diff
@@ -20,20 +20,19 @@ def wrapper(request, *args, **kwargs):
 
         if request.content_type not in ['application/octet-stream', '', 'application/json']:
             return JsonResponse({'error': 'Concent supports only application/octet-stream and application/json.'}, status = 415)
-
-        if len(request.body) == 0:
+        if len(request.body) == 0:  # pylint: disable=len-as-condition
             message = None
         else:
             if request.content_type == '':
                 raise Http400('Content-Type is missing.')
             if request.content_type == 'application/json':
-                message = request.body
+                message = json.loads(request.body.decode('ascii').replace("'", "\""))
```

Comment:
Why are you replacing single quotes with double quotes in the message? This does not look like something that belongs here.